### ISSUE TITLE
PATH_TO_VICUNA Error 

### ISSUE THREAD

**Issue by Shlok-crypto** [DebugML/adversarial_prompting#2]
Hi, @patrickrchao @riceric22 

I was am trying to run your `run_text_exp.py` but I keep getting an error relate to `PATH_TO_VICUNA`
The error is `OSError: PATH_TO_VICUNA is not a local folder and is not a valid model identifier listed on 'https://huggingface.co/models'`

I would really appreciate if could you please look into it. 

Thanks

**Comment by patrickrchao** [comment]
Hi, 
In the `utils/constants.py` folder, you need to update `PATH_TO_VICUNA` located [on line 11](https://github.com/DebugML/adversarial_prompting/blob/master/utils/constants.py#L11) with your proper path to Vicuna weights, you can get them from: [https://github.com/lm-sys/FastChat#vicuna-weights](https://github.com/lm-sys/FastChat#vicuna-weights).

Let me know if this helps!

**Comment by Shlok-crypto** [comment]
Hi Patrick,

Thanks for getting back to me so quickly. I update the `PATH_TO_VICUNA`  with `PATH_TO_VICUNA = "lmsys/vicuna-13b-v1.1"` . 
The code works, but the outputs is  `^C` I changed the parameters (seed text, seed...) still it outputs `^C`

**Comment by patrickrchao** [comment]
Hi Shlok,

Could you clarify what you mean by getting `^C` as output? Additionally, can you share the command you are running into issues with and the entire output?

**Comment by Shlok-crypto** [comment]
this is the command i am running 
`python3 run_text_exp.py --loss_type perplexity --seed 0 --language_model vicuna1.1 --embedding_model opt --seed_text "Explain list comprehension in Python." --wandb_entity "********"` 

![image](https://github.com/DebugML/adversarial_prompting/assets/83566027/2e1eaff3-e244-4686-b64a-0cb5f80e1d52)

**Comment by patrickrchao** [comment]
Thanks for sharing, can you verify that you are using the correct dockerfile (provided in `docker/Dockerfile`)? Additionally, to the best of my knowledge, the `^C` arises from the user interrupting the program (typically `ctrl+c`).

**Comment by Shlok-crypto** [comment]
I am not using the docker. I am running the code on my local machine.  
I have tried with docker, it creates an empty folder `adversarial_bo` inside `workspace`, but that it. I then cloned the git inside it but same error.  

Also I don't get the `^C` error if the language model is `facebook/opt-125m` it seems to happen for all other models especially `vicuna1.1`

I have attached the output for your reference 
![image](https://github.com/DebugML/adversarial_prompting/assets/83566027/21aed0b6-0e5d-4b3d-8d8a-5e21b655ab93)
